### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by nicolinux** [fmalk/whatsapp-to-slack#1]
A little explanation what it does and how to use it would be great!

**Comment by marksyzm** [comment]
I just checked the code and it's just a mail server set up from all I can see

**Comment by cclambie** [comment]
I agree with @marksyzm - there is nothing in the code that suggests it does what the title suggests... 
I am keen to see something like this though, seems doable.
I came across it, as I am looking for a way to set up a Chrome Extension, or Android app, that grabs WhatsApp messages and puts them into a specific other chat program, like slack, that can have multiple users.
Then allows those multiple users to manage the messages more easily.
There is a hole in the WhatsApp communications for small business - only medium to large business is supported for the API... so something like this would be super smart!
